### PR TITLE
Support horizontal/vertical fraction for auto_zoom_on_focus

### PR DESCRIPTION
**Pull request by Jeangowhy**
upgrade zoom_pane(self, auto_zoom) to support auto zoom in X and Y direction，Origami settings update:

// Automatically zoom the active pane.
// Set it to true for the default zoom, or to a user-definable
// fraction of the screen, such as "0.75" or [0.5, 0.7].
"auto_zoom_on_focus": [0.55, 0.75],

### CODE REVIEW

File: origami.py [hunk 1]
```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
         layout = self._on_resize_panes_get_layout(orientation, cells, relevant_index, orig_data, text)
         self.window.set_layout(layout)
 
-    def zoom_pane(self, fraction):
-        if fraction is None:
-            fraction = .9
+    def zoom_pane(self, auto_zoom):
+        # if fraction is None:
+        #     fraction = .9
```
Comment (rchl) [inline origami.py:314]: Don't leave commented out code.

File: origami.py [hunk 2]
```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
...
+        fraction = .9
+        height = .9
+        # if isinstance(auto_zoom, float|int):
+        if isinstance(auto_zoom, float) or isinstance(auto_zoom, int):
+            fraction = auto_zoom
```
Comment (rchl) [inline origami.py:319]: For backward compatibility the specified value should apply to both.

File: origami.py [hunk 3]
```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
...
         self.window.set_layout(layout)
 
-    def zoom_pane(self, fraction):
-        if fraction is None:
-            fraction = .9
+    def zoom_pane(self, auto_zoom):
+        # if fraction is None:
+        #     fraction = .9
+        fraction = .9
+        height = .9
```
Comment (rchl) [inline origami.py:316]: Use better (consistent) naming now. They both specify same type of unit but one is called fraction and one height.

**Review by rchl — CHANGES_REQUESTED** [review]
Missing an update of settings also.

File: origami.py (hunk 3 above)
Comment (Jeangowhy) [reply origami.py]: Thanks you for advice, and I clean some comment code, update some settings.

File: README.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ In the keybindings you can change a `mode` which specifies which separation line
 Automation
 ----------
 
-You can have Origami automatically zoom the active pane by setting `auto_zoom_on_focus` in your Origami user preferences. Set it to `true` for the default zoom, or set it to a user-definable fraction of the screen, such as `0.75`.
+You can have Origami automatically zoom the active pane by setting `auto_zoom_on_focus` in your Origami user preferences. Set it to `true` for the default zoom, or set it to a user-definable fraction of the screen, such as `0.75`, or `[0.5, 0.6]` for horitontal and vertical correspondingly.
```
Comment (rchl) [inline README.md:55]: typo "horitontal"

File: origami.py [hunk 5]
```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
...
-        if fraction is None:
-            fraction = .9
+    
+
+    def zoom_pane(self, auto_zoom):
+        fraction_horizontal = .9
+        fraction_vertical = .9
+        
+        if isinstance(auto_zoom, float) or isinstance(auto_zoom, int):
+            fraction_horizontal = auto_zoom
```
[the remark this answers is not in the document]
Comment (Jeangowhy) [reply origami.py:321]: I don't know clear your mean. Why to set both fractions？

File: Origami.sublime-settings [hunk 6]
```diff
@@ -19,10 +19,15 @@
...
     // fraction of the screen, such as "0.75".
     "auto_zoom_on_focus": false,
+    // "auto_zoom_on_focus": 0.75,
+    // "auto_zoom_on_focus": [0.55, 0.65],
 
     // Automatically close a pane once you've closed the last file in it.
     "auto_close_empty_panes": false,
 
+    // Automatically focus a pane when View.on_hover event.
+    "auto_focus_on_hover": false,
```
Comment (rchl) [inline Origami.sublime-settings:29]: Please don't introduce unrelated changes in this PR.

File: origami.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (rchl) [reply origami.py:321]: Didn't the original code apply specified fraction to both width and height? Here you are applying it only to width, leaving the default `.9` to be used for height.

[the remark this answers is not in the document]
Comment (rchl) [reply origami.py:318]: This is what I'm talking about:

```suggestion
            fraction_horizontal = fraction_vertical = auto_zoom
```

since a single value was previously applying both in horizontal and vertical cases.

(and please let me resolve the issues once I verify that it was addressed)

File: origami.py [hunk 7]
```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
...
+        elif isinstance(auto_zoom, list):
```
[the remark this answers is not in the document]
Comment (rchl) [reply origami.py:319]:
```suggestion
        elif isinstance(auto_zoom, list) and len(auto_zoom) == 2:
```

File: README.md [hunk 8]
```diff
@@ -52,10 +52,11 @@ In the keybindings you can change a `mode` which specifies which separation line
 Automation
 ----------
 
-You can have Origami automatically zoom the active pane by setting `auto_zoom_on_focus` in your Origami user preferences. Set it to `true` for the default zoom, or set it to a user-definable fraction of the screen, such as `0.75`.
+You can have Origami automatically zoom the active pane by setting `auto_zoom_on_focus` in your Origami user preferences. Set it to `true` for the default zoom, or set it to a user-definable fraction of the screen, such as `0.75`, or `[0.5, 0.6]` for horizontal and vertical correspondingly.
 
 Origami can also automatically close a pane for you once you've closed the last file in it. Just set `auto_close_empty_panes` to true in the Origami preferences.
 
+
```
Comment (rchl) [inline README.md:59]: revert spurious blank line

Comment (Jeangowhy) [reply README.md]: Thank you for helping me complete my first PR.✋

File: origami.py [hunk 9]
```diff
@@ -1,4 +1,5 @@
 import sublime
+import sublime_api
```
Comment (rchl) [inline origami.py:2]: remove

**Review by rchl — APPROVED** [review]
